### DISCUSSION TITLE
某些数学符号无法识别

### DISCUSSION THREAD

**Discussion by yhhu99** [stone-zeng/fduthesis#321] [Q&A] [answered]
在使用最新Overleaf模板时，编译报错无法识别控制序列，经排查是无法识别\nsucceq。加上\usepackage{amsthm,amsmath,amssymb}后发现会报很多其他错误，这个需要怎么解决呢？非常感谢！！

**Comment by sikouhjw** [comment, marked as answer]
`unicode-math` 没有这个符号，`unicode-math` 支持符号表查 `unimath-symbols.pdf`
```tex
\documentclass{article}
\usepackage{amsmath,unicode-math}

\begin{document}

\begin{align}
  \nsucc \nsucccurlyeq
\end{align}

\end{document}
```

**Reply by yhhu99** [reply]
Thanks a lot for your kind reply!
